api/services/storeService.js

```diff
@@ -0,0 +1,16 @@
+const { storeDao } = require('../models')
+
+const allStores = async () => {
+  return await storeDao.getAllStores()
+};
+
+const typeStores = async (storeType) => {
+  const storeServiceType = await storeDao.getStoreByType(storeType)
+  return storeServiceType
+}
```

Comment:
```suggestion
const getStoresByType = async (storeType) => {
  return storeDao.getStoreByType(storeType)
}
```
불필요한 변수 선언없이 다음과 같이 작성되면 좋을 것 같습니다😊